### PR TITLE
Feature/609 cta link

### LINKED ISSUE #609: Design: CTA Icon
### User story

A clear and concise description of what the problem is. Eg: As an [actor], I [want|must] [action] so that [achievement].

### Acceptance criteria

Conditions that must be satisfied to be accepted by a user, customer or other stakeholder.

### Definition of Ready

To be completed before the issue can be added to backlog.

- [ ] Acceptance criteria are described
- [ ] Any preconditions are described
- [ ] Possible dependence on external experts is mentioned
- [ ] The story must be understandable for the development team
- [ ] Any requirements for accessibility are described

### Definition of Done

To be completed before the issue can be marked as Done.

- [ ] Code exists on `main` branch
- [ ] Meets acceptance criteria
- [ ] Complies with [the standard of public code](https://standard.publiccode.net/)
- [ ] Story has been sufficiently tested
  - [ ] 90% code coverage
  - [ ] Build completed
  - [ ] Existing tests pass
- [ ] The frontend component complies with [WCAG 2.1 accessibility guidelines](https://www.w3.org/TR/WCAG21/)

### PR DESCRIPTION
**Pull request by paulAcato**
Call to action Link variation;
- Default link;
- Video variant;
- Hovered;
- Focused;
- Grouped;
- Long;

closes #609

### CODE REVIEW

File: components/CtaLink/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "rootDir": "src",
+    "outDir": "dist",
+    "composite": true
+  },
+  "include": ["src/**/*"],
+  "exclude": ["dist", "src/**/*.stories.tsx"],
+  "references": [
+    {
+      "path": "../BaseProps"
+    }
```
Comment (bddjong) [inline components/CtaLink/tsconfig.json:13]: Baseprops are unused

File: proprietary/Components/src/denhaag/cta-link.tokens.json [hunk 2]
```diff
@@ -0,0 +1,69 @@
+{
+  "denhaag": {
+    "cta-link": {
+      "border-color": {
+        "value": "{denhaag.color.grey.2.value}"
```
Comment (bddjong) [inline proprietary/Components/src/denhaag/cta-link.tokens.json:5]: `.value` can be omitted with the latest version of style dictionary.

**Review by bddjong — CHANGES_REQUESTED** [review]
Few small comments

**Comment by robbieatacato** [comment]
added changes requested above

**Pull request merged**